html/index.js

```diff
@@ -923,53 +966,33 @@ function generateListItem (data, item) {
 }
 
 // event listener to support expanding children items on click
-function expand (data, event) {
+function expand (data, elem, event) {
   event.preventDefault()
   event.stopPropagation()
   const et = event.target
   const parent = et.parentElement
   const id = parent.id.replace('item-', '')
   const kids = getChildren(data, id)
   const items = kids.map(generateListItem.bind(null, data))
-  const ul = document.createElement('ul')
   items.forEach(function (li) {
-    ul.appendChild(li)
+    elem.appendChild(li)
   })
-  parent.appendChild(ul)
-  et.classList.remove('plus')
-  et.classList.add('minus')
-  et.textContent = '-'
-  et.addEventListener('click', collapse.bind(null, data), { once: true })
 
   if (event.shiftKey) {
     const max = countChildren(data, id, 0)
     console.log(max)
     initProgressBar()
-    expandAll({ value: ul }, max, { value: 2 })
+    expandAll({ value: elem }, max, { value: 2 })
   }
 }
 
-// event listener for collapsing items
```

Comment:
The fast tree loader is broken. Instead of using sl-tree to beautify the ux, use pure css. Remove all the changes made to the tree and replace with :
https://github.com/test-network-function/parser/compare/main...edcdavid:change-style?expand=1

Reply:
thanks alot that look better with the expand all